common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1896,6 +1899,8 @@ def render(self, block, view_name, context=None):
         integrate it into a larger whole.
 
         """
+        beeline.add_context_field("block", block)
+        beeline.add_context_field("view_name", view_name)
```

Comment:
```suggestion
        beeline.add_context_field("view_name", view_name)
        beeline.add_context_field("xblock_name", self.name)
        beeline.add_context_field("xblock_display_name", getattr(self, 'display_name', 'N/A')
        
        try
            beeline.add_context_field("xblock_usage_id", self.scope_ids.usage_id)
        except AttributeError:
            beeline.add_context_field("xblock_usage_id", "N/A")
```

Related to my comment in Jira: https://appsembler.atlassian.net/browse/RED-1066?focusedCommentId=25754